### DISCUSSION TITLE
"Review changes" button not function in  commits view

### DISCUSSION THREAD

**Discussion by NirBenjamin** [community/community#78558] [Pull Requests]
### Select Topic Area

Bug

### Body

Steps to reproduce:

1. open a PR
2. navigate to 'commits' (https://github.com/<myOrg>/<myRepo>/pull/11/commits)
3. open a commit
4. click (or at least try to click on review changes button - nothing happened...

![image](https://github.com/community/community/assets/18168207/6eed5ccd-d261-4e30-ab03-dfad20d112ba)

after refresh it works.

More info:
browser: Microsoft Edge

**Comment by fraxken** [comment]
I'm also impacted by that bug. However refresh do nothing.

**Comment by Grant1219** [comment]
I'm also having this problem, the `Review changes` button does nothing at all. Same behavior in Firefox and Chrome.

EDIT: After disabling uBlock Origin for Github... the popup to finish the review works again. Give that a try if you're using uBlock, but this wasn't happening before.

**Comment by DylanDevelops** [comment]
This is currently a bug. I would report this to the GitHub feedback page, and they will do their best to fix it.

**Comment by queenofcorgis** [comment]
Thanks everyone for reporting, this is the place to do so, - I've made sure this post has gotten passed along to our team 🤠 .

**Comment by trentp-igor** [comment]
Still see this problem, using Chrome  112.0.5615.121
